02-22 17:27:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Tabelle "USER" nicht gefunden
Table "USER" not found; SQL statement:
insert into User (userMail, userPassword, userName) values (?, ?, ?) [42102-174]
02-22 17:27:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Tabelle "USER" nicht gefunden
Table "USER" not found; SQL statement:
select this_.userName as userName1_0_0_, this_.userMail as userMail2_0_0_, this_.userPassword as userPass3_0_0_ from User this_ where this_.userName=? and this_.userPassword=? [42102-174]
02-22 17:30:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Tabelle "USER" nicht gefunden
Table "USER" not found; SQL statement:
insert into User (userMail, userPassword, userName) values (?, ?, ?) [42102-174]
02-22 17:35:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Tabelle "USER" nicht gefunden
Table "USER" not found; SQL statement:
select this_.userName as userName1_0_0_, this_.userMail as userMail2_0_0_, this_.userPassword as userPass3_0_0_ from User this_ where this_.userName=? and this_.userPassword=? [42102-174]
02-22 17:35:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Tabelle "USER" nicht gefunden
Table "USER" not found; SQL statement:
select this_.userName as userName1_0_0_, this_.userMail as userMail2_0_0_, this_.userPassword as userPass3_0_0_ from User this_ where this_.userName=? and this_.userPassword=? [42102-174]
02-22 17:36:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Tabelle "USER" nicht gefunden
Table "USER" not found; SQL statement:
select this_.userName as userName1_0_0_, this_.userMail as userMail2_0_0_, this_.userPassword as userPass3_0_0_ from User this_ where this_.userName=? and this_.userPassword=? [42102-174]
02-22 17:59:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Tabelle "USER" nicht gefunden
Table "USER" not found; SQL statement:
select this_.userName as userName1_0_0_, this_.userMail as userMail2_0_0_, this_.userPassword as userPass3_0_0_ from User this_ where this_.userName=? and this_.userPassword=? [42102-174]
02-22 18:15:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Das Objekt wurde bereits geschlossen
The object is already closed [90007-174]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:978)
	at org.h2.jdbc.JdbcPreparedStatement.checkClosed(JdbcPreparedStatement.java:1574)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:954)
	at org.h2.jdbc.JdbcStatement.getMaxRows(JdbcStatement.java:356)
	at org.hibernate.engine.jdbc.internal.JdbcCoordinatorImpl.close(JdbcCoordinatorImpl.java:530)
	at org.hibernate.engine.jdbc.internal.JdbcCoordinatorImpl.release(JdbcCoordinatorImpl.java:407)
	at org.hibernate.engine.jdbc.batch.internal.AbstractBatchImpl.releaseStatements(AbstractBatchImpl.java:173)
	at org.hibernate.engine.jdbc.batch.internal.NonBatchingBatch.addToBatch(NonBatchingBatch.java:76)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3124)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3587)
	at org.hibernate.action.internal.EntityInsertAction.execute(EntityInsertAction.java:103)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:453)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:345)
	at org.hibernate.event.internal.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:350)
	at org.hibernate.event.internal.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:56)
	at org.hibernate.internal.SessionImpl.flush(SessionImpl.java:1218)
	at org.hibernate.internal.SessionImpl.managedFlush(SessionImpl.java:421)
	at org.hibernate.engine.transaction.internal.jdbc.JdbcTransaction.beforeTransactionCommit(JdbcTransaction.java:101)
	at org.hibernate.engine.transaction.spi.AbstractTransactionImpl.commit(AbstractTransactionImpl.java:177)
	at startup.Init.initialize(Init.java:38)
	at startup.ServerStartup.main(ServerStartup.java:72)
02-22 21:10:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Das Objekt wurde bereits geschlossen
The object is already closed [90007-174]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:978)
	at org.h2.jdbc.JdbcPreparedStatement.checkClosed(JdbcPreparedStatement.java:1574)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:954)
	at org.h2.jdbc.JdbcStatement.getMaxRows(JdbcStatement.java:356)
	at org.hibernate.engine.jdbc.internal.JdbcCoordinatorImpl.close(JdbcCoordinatorImpl.java:530)
	at org.hibernate.engine.jdbc.internal.JdbcCoordinatorImpl.release(JdbcCoordinatorImpl.java:407)
	at org.hibernate.engine.jdbc.batch.internal.AbstractBatchImpl.releaseStatements(AbstractBatchImpl.java:173)
	at org.hibernate.engine.jdbc.batch.internal.NonBatchingBatch.addToBatch(NonBatchingBatch.java:76)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3124)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3587)
	at org.hibernate.action.internal.EntityInsertAction.execute(EntityInsertAction.java:103)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:453)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:345)
	at org.hibernate.event.internal.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:350)
	at org.hibernate.event.internal.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:56)
	at org.hibernate.internal.SessionImpl.flush(SessionImpl.java:1218)
	at org.hibernate.internal.SessionImpl.managedFlush(SessionImpl.java:421)
	at org.hibernate.engine.transaction.internal.jdbc.JdbcTransaction.beforeTransactionCommit(JdbcTransaction.java:101)
	at org.hibernate.engine.transaction.spi.AbstractTransactionImpl.commit(AbstractTransactionImpl.java:177)
	at startup.Init.initialize(Init.java:38)
	at startup.ServerStartup.main(ServerStartup.java:72)
02-22 21:28:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Das Objekt wurde bereits geschlossen
The object is already closed [90007-174]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:978)
	at org.h2.jdbc.JdbcPreparedStatement.checkClosed(JdbcPreparedStatement.java:1574)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:954)
	at org.h2.jdbc.JdbcStatement.getMaxRows(JdbcStatement.java:356)
	at org.hibernate.engine.jdbc.internal.JdbcCoordinatorImpl.close(JdbcCoordinatorImpl.java:530)
	at org.hibernate.engine.jdbc.internal.JdbcCoordinatorImpl.release(JdbcCoordinatorImpl.java:407)
	at org.hibernate.engine.jdbc.batch.internal.AbstractBatchImpl.releaseStatements(AbstractBatchImpl.java:173)
	at org.hibernate.engine.jdbc.batch.internal.NonBatchingBatch.addToBatch(NonBatchingBatch.java:76)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3124)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3587)
	at org.hibernate.action.internal.EntityInsertAction.execute(EntityInsertAction.java:103)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:453)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:345)
	at org.hibernate.event.internal.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:350)
	at org.hibernate.event.internal.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:56)
	at org.hibernate.internal.SessionImpl.flush(SessionImpl.java:1218)
	at org.hibernate.internal.SessionImpl.managedFlush(SessionImpl.java:421)
	at org.hibernate.engine.transaction.internal.jdbc.JdbcTransaction.beforeTransactionCommit(JdbcTransaction.java:101)
	at org.hibernate.engine.transaction.spi.AbstractTransactionImpl.commit(AbstractTransactionImpl.java:177)
	at startup.Init.initialize(Init.java:38)
	at startup.ServerStartup.main(ServerStartup.java:72)
02-22 21:55:33 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Zeit√ºberschreitung beim Versuch die Tabelle "USER" zu sperren
Timeout trying to lock table "USER"; SQL statement:
select this_.userName as userName1_0_0_, this_.userMail as userMail2_0_0_, this_.userPassword as userPass3_0_0_ from User this_ where this_.userName=? and this_.userPassword=? [50200-174]
02-22 22:02:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Das Objekt wurde bereits geschlossen
The object is already closed [90007-174]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.message.DbException.get(DbException.java:138)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:978)
	at org.h2.jdbc.JdbcPreparedStatement.checkClosed(JdbcPreparedStatement.java:1574)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:954)
	at org.h2.jdbc.JdbcStatement.getMaxRows(JdbcStatement.java:356)
	at org.hibernate.engine.jdbc.internal.JdbcCoordinatorImpl.close(JdbcCoordinatorImpl.java:530)
	at org.hibernate.engine.jdbc.internal.JdbcCoordinatorImpl.release(JdbcCoordinatorImpl.java:407)
	at org.hibernate.engine.jdbc.batch.internal.AbstractBatchImpl.releaseStatements(AbstractBatchImpl.java:173)
	at org.hibernate.engine.jdbc.batch.internal.NonBatchingBatch.addToBatch(NonBatchingBatch.java:76)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3124)
	at org.hibernate.persister.entity.AbstractEntityPersister.insert(AbstractEntityPersister.java:3587)
	at org.hibernate.action.internal.EntityInsertAction.execute(EntityInsertAction.java:103)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:453)
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:345)
	at org.hibernate.event.internal.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:350)
	at org.hibernate.event.internal.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:56)
	at org.hibernate.internal.SessionImpl.flush(SessionImpl.java:1218)
	at org.hibernate.internal.SessionImpl.managedFlush(SessionImpl.java:421)
	at org.hibernate.engine.transaction.internal.jdbc.JdbcTransaction.beforeTransactionCommit(JdbcTransaction.java:101)
	at org.hibernate.engine.transaction.spi.AbstractTransactionImpl.commit(AbstractTransactionImpl.java:177)
	at startup.Init.initialize(Init.java:38)
	at startup.ServerStartup.main(ServerStartup.java:72)
